docs/about/faq.mdx

```diff
@@ -119,6 +119,11 @@ Once the node wakes up from sleep, your phone will relay any delayed messages th
 - Android instructions see: [Android Usage](/docs/software/android/usage#configuration-options)
 - Python CLI instructions see: [Python Usage](/docs/software/python/cli/usage#changing-settings)
 
+### I am in Europe and my device seems to stop transmitting after a while, what is going on? 
+
+Europe has an hourly duty cycle limit of 10% in the frequency band that Meshtastic uses. It might be that you hit this limit if you are sending a lot. You can confirm this by checking whether you see duty cycle limit errors in the serial log or the MeshLog of the iOS app. 
+To limit traffic, you can consider setting the device metrics and position update intervals higher. Alternatively, you can override the duty cycle limit, but then you will violate the regulations if you are not a HAM operator.
```

Comment:
Maybe change to

LIcensed HAM operators can override the duty cycle limit without violating EU regulations.

Reply:
> Maybe change to
> 
> LIcensed HAM operators can override the duty cycle limit without violating EU regulations.

Please don't reference HAM exceptions here, since the 868 MHz Band is NOT a HAM band in Europe. HAM mode will remove the power limits on 433 (and the other bands) but duty cycle override is 'just not being in compliance'. Except again for EU_433 ;-)

```
144,0 - 146,0 MHz | 2m Band | 750W 
430,0 - 440,0 MHz | 70cm Band | 750W
1.240,0 - 1.300,0 MHz | 23cm Band | 750W
2.320,0 - 2.450,0 MHz | 11cm Band | 75W
```

Reply:
Good point. I just removed the notion about HAM exception.